app/helpers/users_helper.rb

```diff
@@ -1,2 +1,3 @@
 module UsersHelper
+  
 end
```

Comment:
I've heard in the past that you end up deleting a ton of stuff from scaffolds, but until now I haven't had a big emphasis on it. I'll be sure to keep an eye on these things in the future, or just write the code myself.